documentation/api/1.0/api.md

```diff
@@ -0,0 +1,864 @@
+	// More info in mknod(1).
+	DevType string
+
+	// Major, minor numbers for device.
+	Major int64
+	Minor int64
+
+	// FileMode permission bits for the device.
+	FileMode os.FileMode
+
+	// id of the device owner.
+	UID uint32
+
+	// id of the device group.
+	GID uint32
+
+	// Hotplugged is used to store device state indicating if the
+	// device was hotplugged.
+	Hotplugged bool
+
+	// ID for the device that is passed to the hypervisor.
+	ID string
+}
+```
+
+#### `VCPod`
+```Go
+// VCPod is the Pod interface
+// (required since virtcontainers.Pod only contains private fields)
+type VCPod interface {
+	Annotations(key string) (string, error)
+	GetAllContainers() []VCContainer
+	GetAnnotations() map[string]string
+	GetContainer(containerID string) VCContainer
+	ID() string
+	SetAnnotations(annotations map[string]string) error
+}
+```
+
+### Pod Functions
+
+* [CreatePod](#createpod)
+* [DeletePod](#deletepod)
+* [StartPod](#startpod)
+* [StopPod](#stoppod)
+* [RunPod](#runpod)
+* [ListPod](#listpod)
+* [StatusPod](#statuspod)
+* [PausePod](#pausepod)
+* [ResumePod](#resumepod)
+
+#### `CreatePod`
+```Go
+// CreatePod is the virtcontainers pod creation entry point.
+// CreatePod creates a pod and its containers. It does not start them.
+func CreatePod(podConfig PodConfig) (VCPod, error)
+```
+
+#### `DeletePod`
+```Go
+// DeletePod is the virtcontainers pod deletion entry point.
+// DeletePod will stop an already running container and then delete it.
+func DeletePod(podID string) (VCPod, error)
+```
+
+#### `StartPod`
+```Go
+// StartPod is the virtcontainers pod starting entry point.
+// StartPod will talk to the given hypervisor to start an existing
+// pod and all its containers.
+func StartPod(podID string) (VCPod, error)
+```
+
+#### `StopPod`
+```Go
+// StopPod is the virtcontainers pod stopping entry point.
+// StopPod will talk to the given agent to stop an existing pod
+// and destroy all containers within that pod.
+func StopPod(podID string) (VCPod, error)
+```
+
+#### `RunPod`
+```Go
+// RunPod is the virtcontainers pod running entry point.
+// RunPod creates a pod and its containers and then it starts them.
+func RunPod(podConfig PodConfig) (VCPod, error)
+```
+
+#### `ListPod`
+```Go
+// ListPod is the virtcontainers pod listing entry point.
+func ListPod() ([]PodStatus, error)
+```
+
+#### `StatusPod`
+```Go
+// StatusPod is the virtcontainers pod status entry point.
+func StatusPod(podID string) (PodStatus, error)
+```
+
+#### `PausePod`
+```Go
+// PausePod is the virtcontainers pausing entry point which pauses an
+// already running pod.
+func PausePod(podID string) (VCPod, error)
+```
+
+#### `ResumePod`
+```Go
+// ResumePod is the virtcontainers resuming entry point which resumes
+// (or unpauses) and already paused pod.
+func ResumePod(podID string) (VCPod, error)
+```
+
+## Container API
+
+The virtcontainers 1.0 container API manages specific pod containers
+[lifecycle](#container-functions).
```

Comment:
I think this should be 'manages a specific', or 'pod container lifecycles' - feels slightly incomplete sentence as is ?